Client/src/Pages/Auth/Login.jsx

```diff
@@ -433,9 +320,9 @@ const Login = () => {
 				setErrors((prev) => ({ ...prev, email: error.details[0].message }));
 				createToast({ body: error.details[0].message });
 			} else {
-				setStep(2);
```

Comment:
@ajhollid I know you requested this change, but this gets a bit confusing to me: we are on the component StepOne, but it is  the step 0. If we are changing to step 0 I would suggest changing the name of the component as well

Reply:
> @ajhollid I know you requested this change, but this gets a bit confusing to me: we are on the component StepOne, but it is the step 0. If we are changing to step 0 I would suggest changing the name of the component as well

Agreed, it is confusing.  I agree changing the name of the component is a good idea anyways, `StepOne` is really not descriptive as to what Step One actually is.

@jennifer-gan can you please change the names when refactoring out the components?  Thank you!

Reply:
sure, fixed here: https://github.com/bluewave-labs/checkmate/commit/c72334253d9bc4db2cf4daf900c975b884984e9e